cstruct.opam

```diff
@@ -12,7 +12,7 @@ build: [
   ["jbuilder" "subst" "-p" name "--name" name] {pinned}
   ["jbuilder" "build" "-p" name "-j" jobs]
 ]
-build-depends: ["jbuilder" "runtest" "-p" name]
+build-test: ["jbuilder" "runtest" "-p" name]
```

Comment:
This is causing the appveyor failure, since the ppx tests are being built when only cstruct is installed. It should be fixable by assigning the `ppx_test` directory to the ppx_cstruct package with something like:

```




(executables
  (package ppx_cstruct)
  ((libraries (cstruct-unix oUnit))
  (preprocess (pps (ppx_cstruct)))
  (names (pcap basic enum))))
```

(note the new `package` field to manually assign it rather than defaulting to `cstruct`).

This should also fail on Linux, but our test coverage isn't quite there yet I think.